aurt/paper_code_and_plots.py

```diff
@@ -0,0 +1,309 @@
+

```

Comment:
Is this file being run as part of the CI? If not, it would be good to make it part of it (with plotting disabled)

Reply:
I think this file was mistakenly added to the repo. We currently use the plot functions defined in robot_calibration. @Daniella1 what do you think?

Reply:
I don't think the file was supposed to be pushed to the repo.